resources/web_app_auth_settings.go

```diff
@@ -0,0 +1,300 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/web/mgmt/2020-12-01/web"
+	"github.com/cloudquery/cq-provider-azure/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func WebAppAuthSettings() *schema.Table {
+	return &schema.Table{
+		Name:        "aws_web_app_auth_settings",
+		Description: "SiteAuthSettings configuration settings for the Azure App Service Authentication / Authorization feature",
+		Resolver:    fetchWebAppAuthSettings,
+		Options:     schema.TableCreationOptions{PrimaryKeys: []string{"app_cq_id", "id"}},
+		Columns: []schema.Column{
+			{
+				Name:        "app_cq_id",
+				Description: "Unique ID of azure_web_apps table (FK)",
+				Type:        schema.TypeUUID,
+				Resolver:    schema.ParentIdResolver,
+			},
+			{
+				Name:        "app_id",
+				Description: "Unique CloudQuery ID of azure_web_apps table (FK)",
```

Comment:
the other way round this is the app_id is the original resource id and the app_cq_id is the unique cloudquery id